src/cgame/cg_rocket_draw.cpp

```diff
@@ -75,6 +75,46 @@ void Rocket_UpdateLanguage()
 		hudContext->Update();
 	updateLanguage = false;
 }
+
+// Rml already implement ElementText so we better not want to
+// name this TextElement to avoid confusion.
+class CustomizableTextElement : public Rml::Element
+{
+public:
+	CustomizableTextElement(const Rml::String& tag) : Rml::Element(tag) {}
+
+private:
+	void OnUpdate() override
+	{
+		int flags = 0;
+
+		if ( originalRML_.empty() )
+		{
+			GetInnerRML(originalRML_);
+			if ( HasAttribute( "quake" ) )
+			{
+				flags |= RP_QUAKE;
+			}
+
+			if ( HasAttribute( "emoticons" ) )
+			{
+				flags |= RP_EMOTICONS;
+			}
+
+			if ( flags != 0 )
```

Comment:
This behavior is not the same: in the first, inner children will be escaped properly and output as such, in the latter, they will be parsed. I think we should run it through Rocket_QuakeToRML in either case.

<text> is basically akin to <pre>.

Reply:
Ah yes, forgot the requirement to escape things.

Reply:
Or the escaping can be a property.

Reply:
Hmm, now I notice that our `translate` tag does not escape when there is no `quake` and `emoticon` property, but we may want to not escape it to be able to translate strings like: `^1you have ^2<credits /> [bsuit]`, with such string the code should not be escaped.

So, maybe we better want to add another flag for escaping or not.

Reply:
Yes it's intentional that you could use a bold tag inside the translation for example.

Reply:
I just think we better rename `Rocket_QuakeToRML` to something more generic at some point, and just make it enable some text processing by observing flags like `RP_EMOTICONS`, `RP_QUAKE` and `RP_ESCAPE`.